google/downloader.sh

```diff
@@ -15,10 +15,6 @@ set -x
 curl -s https://www.gstatic.com/ipranges/goog.txt > /tmp/goog.txt
 curl -s https://www.gstatic.com/ipranges/cloud.json > /tmp/cloud.json
 
-# Public GoogleBot IP ranges
-# From: https://developers.google.com/search/docs/advanced/crawling/verifying-googlebot
```

Comment:
Leave it as it is, don't delete anything here.

Reply:
Okay, I restored it. I hope I understood what you wanted and apologize if I did not.

Reply:
There are still changes to be made here, we need to make the contents of this file (`google/downloader.sh`) to be like in the main branch.

Reply:
I apologize; I did misunderstand, and GitHub's interface is confusing for me.